public/static/docs/command-reference/get-url.md

```diff
@@ -13,49 +12,55 @@ system.
 ## Description
 
 In some cases it's convenient to get a <abbr>data artifact</abbr> from a remote
-location into the current working directory, regardless of whether it's a DVC
-project. The `dvc get-url` command helps the user do just that.
+location into the local file system. The `dvc get-url` command helps the user do
+just that.
 
 The `url` argument should provide the location of the data to be downloaded,
 while `out` can be used to specify the directory and/or file name desired for
 the downloaded data. If an existing directory is specified, then the output will
 be placed inside of it.
 
-Note that this command doesn't require an existing DVC project to run in. It's a
-single-purpose command that can be used out of the box after installing DVC.
-
-> See `dvc get` to download data or model files or directories from other
-> <abbr>DVC repository</abbr> (e.g. GitHub URLs).
+Note that this command doesn't require an existing <abbr>DVC project</abbr> to
+run in. It's a single-purpose command that can be used out of the box after
+installing DVC.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type    | Discussion     | URL format                                 |
+| Type    | Description    | `url` format                               |
 | ------- | -------------- | ------------------------------------------ |
-| `local` | Local path     | `/path/to/local/file`                      |
-| `s3`    | Amazon S3      | `s3://mybucket/data.csv`                   |
-| `gs`    | Google Storage | `gs://mybucket/data.csv`                   |
-| `ssh`   | SSH server     | `ssh://user@example.com:/path/to/data.csv` |
-| `hdfs`  | HDFS           | `hdfs://user@example.com/path/to/data.csv` |
-| `http`  | HTTP to file   | `https://example.com/path/to/data.csv`     |
+| `local` | Local path     | `/path/to/local/data`                      |
+| `s3`    | Amazon S3      | `s3://mybucket/data`                       |
+| `gs`    | Google Storage | `gs://mybucket/data`                       |
+| `ssh`   | SSH server     | `ssh://user@example.com:/path/to/data`     |
+| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/path/to/data.csv` |
+| `http`  | HTTP to file\* | `https://example.com/path/to/data.csv`     |
+
+> \* HDFS and HTTP **do not** support downloading entire directories, only
+> single files.
+
+<!-- Separate MD quote: -->
 
 > If you installed DVC via `pip` and plan to use cloud services as remote
 > storage, you might need to install these optional dependencies: `[s3]`,
 > `[azure]`, `[gdrive]`, `[gs]`, `[oss]`, `[ssh]`. Alternatively, use `[all]` to
 > include them all. The command should look like this: `pip install "dvc[s3]"`.
 > (This example installs `boto3` library along with DVC to support S3 storage.)
 
-Another way to understand the `dvc get-url` command is as a tool for downloading
-data files.
+<!-- Separate MD quote: -->
+
+> See `dvc get` to download data/model files or directories from other <abbr>DVC
```

Comment:
three notes in a raw - smells bad )

Reply:
Agree. And import-url has 4 in a row... Will think about this ⏳

Reply:
Addressed in 1551b4b9 (part of #966).